inputremapper/injection/macros/macro.py

```diff
@@ -469,10 +469,11 @@ def add_event(self, type_: Union[str, int], code: Union[str, int], value: int):
         self.tasks.append(lambda handler: handler(type_, code, value))
         self.tasks.append(self._keycode_pause)
 
-    def add_mouse(self, direction: str, speed: int):
+    def add_mouse(self, direction: str, speed: int, acceleration: Optional[float] = None):
```

Comment:
To make `black` pass (1/2):

```suggestion
    def add_mouse(
        self,
        direction: str,
        speed: int,
        acceleration: Optional[float] = None,
    ):
```

To get `black` to put the parameter below each other, the trailing comma on the last parameter can be added.